extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -209,16 +209,16 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 	public _perfDataCollectionErrors!: string[];
 	public _perfDataCollectionIsCollecting!: boolean;
 
-	public readonly _performanceDataQueryIntervalInSeconds = 5;		// TO-DO: update value
-	public readonly _staticDataQueryIntervalInSeconds = 30;			// TO-DO: update value
-	public readonly _numberOfPerformanceDataQueryIterations = 11;	// TO-DO: update value
+	public readonly _performanceDataQueryIntervalInSeconds = 30;
+	public readonly _staticDataQueryIntervalInSeconds = 60;
+	public readonly _numberOfPerformanceDataQueryIterations = 19;
```

Comment:
@neetusingh-ms, just a heads up - production DMA/Azure Migrate queries data every 30 seconds for 20 iterations, which equals exactly 10 minutes. However, in this case, since we run the data collection and the actual SKU recommendations on separate timers, if they're both 10 minutes then there's a possibility that it will try to generate SKU recommendations before the data is done being written. So let's drop the number of iterations from 20 down to 19 to ensure the data gets written in time, and we can come up with a more robust IO solution in the future

Reply:
Nit: a TODO for iteration update.